.github/workflows/apero-docs-preview.yaml

```diff
@@ -0,0 +1,42 @@
+on:
+  push:
+    branches:
+      - gha
+
+name: build-site
+
+jobs:
+  build-site:
+    runs-on: macOS-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      
+      - uses: r-lib/actions/setup-r@v1
+
+      - name: Install dependencies
+        run: |
+          install.packages("blogdown")
+          blogdown::install_hugo("0.79.0", extended=TRUE, force=TRUE)
+          blogdown::install_theme(theme="hugo-apero/hugo-apero", force=TRUE)
+        shell: Rscript {0}
+
+      - name: Build website
+        run: |
+          git config --local user.email "actions@github.com"
+          git config --local user.name "GitHub Actions"
```

Comment:
Not sure you need this for this step. 

Is it for the step after when committing to github pages ?

Reply:
Interesting! I had issues over the summer that maybe have changed? https://github.com/r-lib/pkgdown/issues/1329#issuecomment-659444918

Reply:
With pkgdown, I believe this is because the deploy functions requires git to be set up. 

But here you are just building the hugo site and then deploying using the action (that I believe does not need this config as it has its own [way](https://github.com/JamesIves/github-pages-deploy-action#optional-choices))

I may be wrong though. It seems just unnecessary to config this if we don't use git.

Reply:
Ah! Let's see 😅

Reply:
Seems to work ? :)